### DISCUSSION TITLE
Add cooldowns so other items can be purchased after purchasing something

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#599] [Suggestions]
It would be a great feature to include logic that would prevent users from being able to purchase Pack B after Pack A has been purchased, until the cooldown for Pack A's purchase expires. A practical real-life example would be if a player purchases a "24hr Server Boost for Harvesting" - which swaps out server configs for one with dynamic configs. If Player B purchases "24hr Server Boost for Experience," that would cancel out the Harvesting boost with overwriting it with a new config. As a solution to this, I propose having logic in place that would prevent the "Experience Pack" from being purchased if a cooldown for a "Harvesting Pack" (or whatever these packs may be named) is active.

**Comment by mvaessen** [comment]
Hi, can you please post the second suggestion on its own suggestion post please
